catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -123,6 +159,27 @@ public void testInPlaceMonitoring()
     }
 
     @Test
+    public void testInPlaceDavMonitoring()
+            throws IOException, InterruptedException, SourceUnavailableException, IngestException,
```

Comment:
I prefer the ease/brevity of the Exception for tests but there isn't anything wrong with listing them out 👍